astroquery/conftest.py

```diff
@@ -1,78 +0,0 @@
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
-import os
-from pathlib import Path
-
-from astropy.utils import minversion
-import numpy as np
-import pytest
-# this contains imports plugins that configure py.test for astropy tests.
-# by importing them here in conftest.py they are discoverable by py.test
-# no matter how it is invoked within the source tree.
-
-from pytest_astropy_header.display import (PYTEST_HEADER_MODULES,
-                                           TESTED_VERSIONS)
-
-
-# Keep this until we require numpy to be >=2.0
-if minversion(np, "2.0.0.dev0+git20230726"):
-    np.set_printoptions(legacy="1.25")
-
-
-def pytest_configure(config):
-    config.option.astropy_header = True
-
-
-# Add astropy to test header information and remove unused packages.
-# Pytest header customisation was introduced in astropy 1.0.
-
-try:
-    PYTEST_HEADER_MODULES['Astropy'] = 'astropy'
-    PYTEST_HEADER_MODULES['regions'] = 'regions'
-    PYTEST_HEADER_MODULES['pyVO'] = 'pyvo'
-    PYTEST_HEADER_MODULES['mocpy'] = 'mocpy'
-    PYTEST_HEADER_MODULES['astropy-healpix'] = 'astropy_healpix'
-    PYTEST_HEADER_MODULES['vamdclib'] = 'vamdclib'
-
-    # keyring doesn't provide __version__ any more
-    # PYTEST_HEADER_MODULES['keyring'] = 'keyring'
-    del PYTEST_HEADER_MODULES['h5py']
-    del PYTEST_HEADER_MODULES['Scipy']
-    del PYTEST_HEADER_MODULES['Pandas']
-except (NameError, KeyError):
-    pass
-
-# add '_testrun' to the version name so that the user-agent indicates that
-# it's being run in a test
-from . import version
-version.version += '_testrun'
```

Comment:
Actually, I think this was really useful when we look at our logs upstream so that we get the usage statistics right. Could it be kept?

Reply:
This should all be the case, this PR just removed the duplicated conftest file, but the remaining one on the top level should still contain all the useful bits and pieces.